packages/components/.eslintrc.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     'react'],
   "ignorePatterns": ["*.scss.d.ts", '*.snap'],
   'rules': {
+    "space-before-function-paren": [2, "never"],
```

Comment:
@lebaphi - is the above change going to cause lots of linter updates in other files?

Reply:
Don't worry @IanMayo, it just disables this rule, doesn't reformat other files

Reply:
I cleaned up some code and removed it. So just ignore it.